app/assets/javascripts/discourse/components/emoji-picker.js.es6

```diff
@@ -58,6 +58,9 @@ export default Ember.Component.extend({
       this._scrollTo();
       this._updateSelectedDiversity();
       this._checkVisibleSection(true);
+
+      if (!this.site.isMobileDevice || this.get("isEditorFocused"))
+        this.$filter.find("input[name='filter']").focus();
```

Comment:
I think we also need to test for !safariHacksDisabled() in utils, otherwise stuff can get bouncy on iPad